python/perspective/perspective/core/data/pd.py

```diff
@@ -78,6 +78,12 @@ def deconstruct_pandas(data, kwargs=None):
                 if isinstance(v, pd.CategoricalDtype):
                     data[k] = data[k].astype(str)
 
+    # convert StringDtype to str
+    if isinstance(data, pd.DataFrame) and hasattr(pd, "CategoricalDtype"):
+        for k, v in data.dtypes.items():
+            if isinstance(v, pd.StringDtype):
```

Comment:
I think maybe this was meant to be used in the `hasattr`, `hasattr(pd, "StringDType")` since `StringDType` is the new type